services/roles/ntfy/templates/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  ntfy:
+    image: binwiederhier/ntfy
+    container_name: ntfy
+    ports:
+      - "{{ ntfy_port }}:80"
```

Comment:
What is this variable? Don't expose any ports, expose only while testing in local machine.